scripts/generic-ci-tests.sh

```diff
@@ -162,7 +162,7 @@ case "$TEST_SUITE" in
 
     "bok-choy")
 
-        PAVER_ARGS="-n $NUMBER_OF_BOKCHOY_THREADS --with-flaky --with-xunit"
+        PAVER_ARGS="-n $NUMBER_OF_BOKCHOY_THREADS"
```

Comment:
Unlike nose, pytest doesn't need a separate command line flag to enable support for the flaky package; if it's installed, it just works.  https://github.com/box/flaky#activating-the-plugin